### DISCUSSION TITLE
iOS崩溃｜版本4.2.41｜谢谢

### DISCUSSION THREAD

**Discussion by colourfulStar** [Tencent/libpag#1784] [Q&A] [answered]
![image](https://github.com/Tencent/libpag/assets/14305608/3fd84b56-db74-4cbf-ba1c-83ec1c51930f)
![image](https://github.com/Tencent/libpag/assets/14305608/dd7b3242-dc99-4d1c-bec5-35c1a679be80)
请问更新sdk到最新版本可以解决问题吗？

**Comment by kevingpqi123** [comment, marked as answer]
先升级至最新版本
